frontend/src/wordInteractions.js

```diff
@@ -53,21 +82,64 @@ function playerTypesWord () {
   typingForm.addEventListener('submit', (event) => {
     event.preventDefault();
     const typedSubmission = typingForm['player-input'].value;
-    WORDS_TYPED.push(typedSubmission);
-    if (WORDS_SEEN.includes(typedSubmission)) {
-      removeWordFromPage(typedSubmission);
+    const matchOnPage = document.getElementById(typedSubmission);
+
+    if (removeWordFromPage(typedSubmission)) {
+      game.wordsTyped.push(typedSubmission);
     } else {
-      console.log('no match on page');
+      game.typos++;
+    }
+    if (game.typos > 2) {
+      gameOver(false);
```

Comment:
This feels like a misnomer. Maybe 'playerWins' instead? This will end the game no matter what, what is true/false is if the player won the game

Reply:
Yeah we can change, gameOver() just means the game ends, and if you look at the function itself I named the paramater passed in win. So win is either set to true or false, but I guess looking at it this way can be confusing.